deploy_polygon/05_catalyst/900_deploy_faucets.ts

```diff
@@ -0,0 +1,20 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {skipUnlessTestnet} from '../../utils/network';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+  const {deployer} = await getNamedAccounts();
+  await deploy('AttributesFaucets', {
+    contract: 'Faucets',
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+  });
+};
+
+func.tags = ['AttributesFaucets', 'AttributesFaucets_deploy'];
```

Comment:
We might want to add an 'L2' tag

Reply:
Thanks, it's already on `deploy_polygon` folder, is it still necessary?

Reply:
it's not